src/index.tsx

```diff
@@ -26,7 +26,8 @@ serviceWorker.unregister()
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals()
+// eslint-disable-next-line no-console
+reportWebVitals(x => console.log('reportWebVitals', x))
```

Comment:
You can avoid the eslint issue by using `console.info` or `console.debug`.

Reply:
```suggestion
reportWebVitals(x => console.debug('reportWebVitals', x))
```
good catch!

Reply:
huh, doesn't look like it liked that. still, debug log level is the way to go. (and we can't use process.env.NODE_ENV b/c this is in the browser)

Reply:
react adds in process.env as a way to get environment variables that were compiled into the app.

Reply:
ok so `debug` still gets the eslint error but `console.info`, `console.warn` and `console.error` do not.